### ISSUE TITLE
No suggestions when typing search entries (Sailfish X)

### ISSUE THREAD

**Issue by vpodzime** [M4rtinK/modrana#241]
When typing the search entries the keyboard doesn't provide word completions/suggestions (at least on Sailfish X). However, the suggestions may be really handy as lot of the search terms are actually in respective language's dictionaries.

**Comment by M4rtinK** [comment]
Yeah, that's definitely a bug - I likely need to set some hint for the entry field to make the text prediction work for the text field - I'll look into that.

BTW, the search field logic really needs also other improvements, such as showing last-N searched terms for easy reuse of past entries.
